plugins/woocommerce-beta-tester/src/live-branches/hooks/live-branches.tsx

```diff
@@ -16,31 +16,44 @@ export type Branch = {
 	install_status: PluginStatus;
 };
 
+type LiveBranchesResponse = {
+	pr: { [ key: string ]: Branch };
+	master: Branch;
+};
+
 export const useLiveBranchesData = () => {
 	const [ branches, setBranches ] = useState< Branch[] >( [] );
 	const [ loading, setLoading ] = useState< boolean >( true );
+	const [ isError, setIsError ] = useState< boolean >( false );
 
 	useEffect( () => {
 		const getBranches = async () => {
-			const res = await apiFetch( {
-				path: `${ API_NAMESPACE }/live-branches/manifest/v1`,
-				method: 'GET',
-			} );
-
-			setBranches(
-				// @ts-ignore
-				Object.entries( res.pr ).map( ( [ , value ] ) => {
-					return value;
-				} ) as Branch[]
-			);
-
-			setLoading( false );
+			try {
+				const res = await apiFetch< LiveBranchesResponse >( {
+					path: `${ API_NAMESPACE }/live-branches/manifest/v1`,
+					method: 'GET',
+				} );
+
+				const prBranches = Object.entries( res.pr ).map(
+					( [ , value ] ) => {
+						return value;
+					}
+				) as Branch[];
+
+				console.log( res.master, prBranches );
```

Comment:
The `console.log` should probably go?

Reply:
Whoops! Thanks for spotting this!

Reply:
Resolved https://github.com/woocommerce/woocommerce/pull/38536/commits/1d75ec22b7bf0c714143a561afed8862a65fa9bd